contrib_bots/bot_lib.py

```diff
@@ -56,78 +56,76 @@ def send_message(self, *args, **kwargs):
                           ' its own messages?')
             sys.exit(1)
 
-def run_message_handler_for_bot(lib_module, quiet, config_file):
-    # Make sure you set up your ~/.zuliprc
-    client = Client(config_file=config_file)
-    restricted_client = BotHandlerApi(client)
+    def run_message_handler_for_bot(self, lib_module, quiet, client):
+        # Make sure you set up your ~/.zuliprc
```

Comment:
@abhijeetkaur this has to be moved to run.py, as it documents the `config_file` param.